dialogue/TownNPCs/BenchGirlRight/BenchGirlRight1.json

```diff
@@ -13,12 +13,17 @@
       "Parece estar bastante enfadada por que no dejo de sacar seises.",
       "Y yo me pregunto... ¿cuando se dará cuenta de que todos los lados tienen un seis?, <w>¡Jiji!</w>"
     ],
+    "pl": [
+      "Wydaje mi się, że ona może być troszkę zdenerwowana tym, że wciąż wyrzucam szóstkę na kości.",
```

Comment:
Is "wyrzucam" the best word to use here? It implies throwing something away, would "rzucam" be a better fit?

Reply:
Actually both words are correct, but I think "wyrzucam" would fit a little bit better. However I can change it if you want.

Reply:
I've always heard "rzucić" being used for rolling a die, but if "wyrzucić" is also an acceptable and regular verb, then that's ok.

Reply:
I asked my friend about this and he also said that "wyrzucić" could be used here, so I think it's correct.